### PR TITLE
Issue #10: Remove obsolete expensive collectors, simplify usage

### PR DESCRIPTION
**Pull request by fgm**
PR for #10

### CODE REVIEW

File: doc/NodeCounters.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# NodeCounter
+## Per-counter
+### Cheap
```
Comment (yched) [inline doc/NodeCounters.md]: The "Cheap" and "Costly" sections in doc/NodeCounters.md are obsolete now ?

File: doc/NodeCounters.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+1. cpuPerSecond: CPU milliseconds used by process during last quasi-second.
+2. cpuPerTickAvg: Average CPU milliseconds used by process per tick during last quasi-second (= 1./4.)
+3. cpuPerTickMax: Maximum of CPU milliseconds used by process since last fetch, not last quasi-second.
+4. tickCount: Exact tick count during last quasi-second.
+5. tickLagAvg:Average tick duration deviation from 1 msec (in msec) during last quasi-second.
+6. tickLagMax:Maximum tick duration deviation from 1 msec (in msec) since last fetch, not last quasi-second.
+  -> use ELS, same accuracy, way cheaper
+7. ticksPerSec:Average ticks per second during last quasi-second.
+
+### ELS
```
Comment (yched) [inline doc/NodeCounters.md]: The "ELS" section lacks the cpuUsageMax and loopCountPerSecSinceLastFetch metrics added in the branch ?

**Review by yched — CHANGES_REQUESTED** [review]
The NodeCounters.md doc added here doesn't seem to account for the changes made in the branch ;-)

File: doc/NodeCounters.md (hunk 1 above)
Comment (fgm) [reply doc/NodeCounters.md]: Done

File: doc/NodeCounters.md (hunk 2 above)
Comment (fgm) [reply doc/NodeCounters.md]: Done

**Pull request merged**